File: UI/Harmonic/Harmonic/ViewController.m
//
//  ViewController.m
//  Harmonic
//
//  Created by gaurav on 02/03/17.
//  Copyright © 2017 gaurav. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


- (IBAction)btn:(id)sender {
    NSString * a=_txt.text;
    int b=[a intValue];
    double harmonicNum=0;
    if (b==0)
        [self.lbl setText:@"Enter Valid Num"];
    
    else{
        for(int i=1;i<=b;i++)
        {
            harmonicNum=harmonicNum+(1.0f/i);
        }
        [self.lbl setText: [NSString stringWithFormat:@"%0.4f",harmonicNum]];
    }
    
    
    
    
}
@end

File: Swift/FirstSwift/FirstSwift/ViewController.swift
//
//  ViewController.swift
//  FirstSwift
//
//  Created by gaurav on 14/03/17.
//  Copyright © 2017 Alk. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    
    @IBOutlet weak var textfield1: UITextField!
    
    @IBOutlet weak var label1: UILabel!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

    @IBAction func PressedButton(_ sender: Any) {
       // label1.text =
        var a = textfield1.text
        label1.text = a
        
    }

}

